src/Components/TacticsPage.js

```diff
@@ -0,0 +1,305 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, Text, Dimensions, Modal, TouchableHighlight, FlatList } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+
+import VimeoVideo from './VimeoVideo';
+
+import theme from '../styles/theme.style';
+
+const screenDimension = Dimensions.get('window');
+
+const TacticsPage = () => {
```

Comment:
I did not change anything to this page because it is the same as EssentialPage. I think we should tweak it at the end when we're ok with everything else